spec/support/schemas/payment_information_errors.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
looks to be the same as `letters_errors.json` could we have one `evss_errors.json` schema?

Reply:
👍